hbp_knowledge/tardbp/ayala2008.bel

```diff
@@ -76,37 +76,50 @@ DEFINE ANNOTATION Cells         AS PATTERN "^BTO:\d{7}$"
 SET Citation = {"PubMed", "18305152"}
 
 SET Evidence = "TDP-43 inhibits exon recognition during splicing upon recruitment to the 3' splice site of the cystic fibrosis transmembrane conductance regulator (CFTR) and apolipoprotein AII transcripts via a sequence of GU repeats (2–5)."
+SET Confidence = "High"
 act(p(HGNC:TARDBP), ma(GO:"GU repeat RNA binding")) -> composite(p(HGNC:TARDBP), p(HGNC:CFTR))
 act(p(HGNC:TARDBP), ma(GO:"GU repeat RNA binding")) -> composite(p(HGNC:TARDBP), p(HGNC:APOA2))
 act(p(HGNC:TARDBP), ma(GO:"GU repeat RNA binding")) -| complex(GO:"exon-exon junction complex")
 complex(GO:"exon-exon junction complex") -> bp(GO:"RNA splicing")
 act(p(HGNC:TARDBP), ma(GO:"GU repeat RNA binding")) -| bp(GO:"RNA splicing")
+UNSET Confidence
 
 SET Evidence = "TDP-43 also has been implicated in the transcription regulation of HIV and the spermatid-specific gene SP-10 through promoter association (7, 8)."
+SET Confidence = "High"
 SET MeSHDisease = "HIV Infections"
 act(p(HGNC:TARDBP), ma(GO:"core promoter binding")) reg bp(GO:"viral transcription")
 UNSET MeSHDisease
+UNSET Confidence
 
 SET Evidence = "More recently, TDP-43 was identified as the main ubiquitinated component of cytoplasmic inclusions in neurodegenerative diseases, specifically frontotemporal lobar degeneration (FTLD) and amyotrophic lateral sclerosis (ALS) (9, 10)."
+SET Confidence = "High"
 SET MeSHDisease = {"Frontotemporal Lobar Degeneration", "Amyotrophic Lateral Sclerosis"}
 p(HGNC:TARDBP) -> p(MESH:"Inclusion Bodies", pmod(Ub))
 UNSET MeSHDisease
 path(OMIT:0026893) isA path(EFO:"neurodegenerative disease")
 path(EFO:"amyotrophic lateral sclerosis") isA path(EFO:"neurodegenerative disease")
+UNSET Confidence
 
 SET Evidence = "Abnormal aggregation of TDP-43 in the cytoplasm now is thought to define a class of frontotemporal dementias termed TDP-43 proteinopathies."
+SET Confidence = "High"
 p(CONSO:"TDP-43 aggregates", loc(GO:cytoplasm)) -- path(DOID:"frontotemporal dementia")
 p(CONSO:"TDP-43 aggregates", loc(GO:cytoplasm)) -- path(MESH:"TDP-43 Proteinopathies")
 path(MESH:"TDP-43 Proteinopathies") isA path(DOID:"frontotemporal dementia")
+UNSET Confidence
 
 #experiment performed on HeLa cells
 
 SET CellLine = "HeLa"
 
 SET Evidence = "The data obtained indicated altered levels of several cell proliferation factors in TDP-43-silenced cells."
+SET Confidence = "High"
 p(HGNC:TARDBP) reg bp(MESH:"Cell Proliferation")
+UNSET Confidence
 
 SET Evidence = "Table 1 lists 16 of these proteins whose functions have been associated with retinoblastoma protein (pRb) activity."
+SET Confidence = "Low"
+#this Evidence should be removed
+#why do? because the table had such information I had to encode it.
```

Comment:
@trushaadeshara8 let me know about this statement.

Reply:
No because we discussed in one of the meetings that unless the statements are in form of statements, we will not encode the table contents.

Reply:
It was discussed that if there is relevant information in tables then we need to search for statements that describe the table or statements that refer to the table.